packages/core/src/components/Modal/private/InnerContent.tsx

```diff
@@ -23,6 +24,10 @@ export type ModalInnerContentProps = {
   subtitle?: React.ReactNode;
   /** Dialog header title. */
   title?: React.ReactNode;
+  /** Top bar, above dialog header. */
+  topBar?: React.ReactNode;
+  /** Whether the top bar content is centered. */
+  topBarCentered?: boolean;
```

Comment:
Couldn't they manage this themselves? `topBar={<Text centerAlign>Sup???</Text>}`

Reply:
no - because extra padding on the left needs to be added so the space that the top bar content goes into has equal distance on the sides:

<img width="686" alt="Screen Shot 2020-04-20 at 4 11 25 PM" src="https://user-images.githubusercontent.com/306275/79808130-bd38d800-8321-11ea-86c7-7cfaa57c0128.png">

so it can be truly centered with the modal's width and not collide with the close button

Reply:
If the close button was position absolutely (instead of using `Row`), that may not be necessary.

Reply:
tried that, but still need the padding to make sure the content doesn't go behind the close button (or over it)